avocado/utils/pci.py

```diff
@@ -82,11 +82,11 @@ def get_num_interfaces_in_pci(dom_pci_address):
 
 def get_disks_in_pci_address(pci_address):
     """
-    Gets disks in a pci_address.
+    Gets disks in a PCI address.
```

Comment:
Well here it's suppose to correlate with the `pci_address` variable. I mean it's parsable both ways, but I'd slightly prefer keeping the `pci_address`.